### PR TITLE
LKE-4205 Please reindex after import

### PR DESCRIPTION
**Pull request by renato-rg**
JIRA:  https://linkurious.atlassian.net/browse/LKE-4205

### CODE REVIEW

File: src/backend/shared.ts [hunk 1]
```diff
@@ -39,6 +39,9 @@ export class CSVUtils {
       return {error: 'Header value is empty'};
     }
 
+    // Remove the headers
+    parseResult.data.shift();
```
Comment (renato-rg) [inline src/backend/shared.ts:43]: there was a bug where the header was duplicated in parsedCSV.headers and in the parsedCSV.records[0]

File: src/backend/graphItemService.ts [hunk 2]
```diff
@@ -260,7 +260,7 @@ export class GraphItemService {
 
     // Parse row by row
     for (let i = 0; i < parsedCSV.records.length; i++) {
-      let [from, to, ...propertyValues] = parsedCSV.records[i - 1];
+      let [from, to, ...propertyValues] = parsedCSV.records[i];
```
Comment (renato-rg) [inline src/backend/graphItemService.ts:263]: this was a bug, in the first iteration record[-1] is undefined

File: src/public/cards/entityProperties.ts [hunk 3]
```diff
@@ -29,8 +29,8 @@ export class CSVEntityProperties {
     this.hideCard();
   }
 
-  setTitle(entityType: EntitiesTypes, headers: string[]) {
-    if (headers.length > 0) {
+  setTitle(entityType: EntityType, headers: string[]) {
+    if (headers.length > (entityType === EntityType.NODE ? 0 : 2)) {
```
Comment (renato-rg) [inline src/public/cards/entityProperties.ts:33]: this was a bug when uploading edges, now it's fixed

File: src/public/models.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
-export enum EntitiesTypes {
-    NODES,
-    EDGES
+export enum EntityType {
+    NODE,
+    EDGE
```
Comment (renato-rg) [inline src/public/models.ts:3]: for consistency with LKE I renamed this to EntityType NODE EDGE :)

File: src/public/cards/entityProperties.ts (hunk 3 above)
Comment (aminerhanemi) [reply src/public/cards/entityProperties.ts:33]: can we add a comment why 0 for node and 2 for edge

File: src/public/cards/importFeedback.ts [hunk 5]
```diff
@@ -45,7 +45,7 @@ export class CSVImportFeedback {
     switch (total) {
       case feedback.success:
         this.fillImportFeedback(
-          `All ${total} ${item} have been imported`,
+          `All ${total} ${item} have been imported. Please index the data-source to make them searchable`,
```
Comment (aminerhanemi) [inline src/public/cards/importFeedback.ts:48]: in the specs when there is only 1 item nodes/edges is singular

File: src/public/cards/importFeedback.ts [hunk 6]
```diff
@@ -61,12 +61,13 @@ export class CSVImportFeedback {
       default :
         const error = feedback.failed === 1 ? 'error' : 'errors';
         this.fillImportFeedback(
-          `${feedback.failed}/${total} ${item} have not been imported due to the following ${error}`,
+          `Only ${feedback.success}/${total} ${item} have been imported. Please index the data-source to make them searchable.<br><br>` +
+            `While ${feedback.failed}/${total} ${item} have <i>not</i> been imported due to the following ${error}`,
```
Comment (aminerhanemi) [inline src/public/cards/importFeedback.ts:65]: same for item

**Review by aminerhanemi — CHANGES_REQUESTED** [review]

**Review by aminerhanemi — APPROVED** [review]

**Pull request merged**